Build error in master from Webpack 5

## Description
After an attempt to release 3.0a1, the resulting CI build is showing this error:

```
 ModuleNotFoundError: Module not found: Error: Can't resolve 'url' in 'C:\Users\RUNNER~1\AppData\Local\Temp\tmp3i4nkjrs\staging\node_modules\sanitize-html\dist'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need these module and configure a polyfill for it.

If you want to include a polyfill, you need to install 'url'.
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.alias: { "url": false }
```

--- a/buildutils/src/ensure-repo.ts
+++ b/buildutils/src/ensure-repo.ts
@@ -29,7 +29,8 @@ const MISSING: Dict<string[]> = {
 };
 
 const UNUSED: Dict<string[]> = {
-  '@jupyterlab/apputils': ['@types/react'],
+  // url is a polyfill for sanitize-html
+  '@jupyterlab/apputils': ['@types/react', 'url'],
   '@jupyterlab/application': ['@fortawesome/fontawesome-free'],
   '@jupyterlab/apputils-extension': ['es6-promise'],
   '@jupyterlab/services': ['node-fetch', 'ws'],

--- a/buildutils/src/webpack-plugins.ts
+++ b/buildutils/src/webpack-plugins.ts
@@ -84,6 +84,9 @@ export namespace WPPlugin {
       callback: any,
       callbackUndelayed: any
     ) {
+      files = Array.from(files);
+      dirs = Array.from(dirs);
+
       const notIgnored = (path: string) => !this.ignored(path);
 
       const ignoredFiles = files.filter(this.ignored);

--- a/packages/apputils/package.json
+++ b/packages/apputils/package.json
@@ -58,7 +58,8 @@
     "@types/react": "~16.9.16",
     "react": "~16.9.0",
     "react-dom": "~16.9.0",
-    "sanitize-html": "~1.20.1"
+    "sanitize-html": "~1.20.1",
+    "url": "^0.11.0"
   },
   "devDependencies": {
     "@jupyterlab/testutils": "^3.0.0-alpha.1",

--- a/packages/apputils/src/sanitizer.ts
+++ b/packages/apputils/src/sanitizer.ts
@@ -1,5 +1,8 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
+
+// sanitize-html uses the url package, so we depend on a standalone version of
+// it which acts as a polyfill for browsers.
 import sanitize from 'sanitize-html';
 
 export interface ISanitizer {

